Production_scripts/Process_extracts/Produce_extract_tests/extract_chi_ltc_tests.R

```diff
@@ -0,0 +1,93 @@
+####################################################
+# Name of file - extract_chi_ltc_tests.R
+# Original Authors - Bateman McBride
+# Original Date - August 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               LTC file received from IT.
+#####################################################
+
+year <- "1920"
+
+new_data <- haven::read_sav(get_ltcs_path(year))
+
+# Find and flag any duplicate chis and chi/postcode combinations
+duplicates <- new_data %>%
+  dtplyr::lazy_dt() %>%
+  dplyr::group_by(chi) %>%
+  dplyr::mutate(duplicate_chi = dplyr::n() > 1) %>%
+  dplyr::ungroup() %>%
+  dplyr::group_by(chi, postcode) %>%
+  dplyr::mutate(duplicate_chi_postcode = dplyr::n() > 1) %>%
```

Comment:
I'm not sure what `duplicate_chi_postcode` is / would tell us? If there's a duplicate CHI, that's an issue, I'm not sure if we care if those duplicate CHIs have the same postcode though?